portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,142 @@
+    <div id="kc">
+        <h2><center>KUN CHEN</center></h2>
+        <p>734-496-3088</p>
+        <p>kunchenkc7@gmail.com | Ann Arbor, MI</p>
+        <p>U of M Class of 2024</p>
+        <p>Bachelor of Science in Computer Science</p>
+        <p>Bachelor of Arts in Economics</p>
+    </div>
+      <div id="aboutme">
+    <h2><center>ABOUT ME</center></h2>
+    <a href="photo.html">
+    <img src="Image/2.jpg" 
+    width="300"
+    height="400"
+    align="center" />
+</a>
+      <p>Name: Kun Chen</p>
+      <p>English Name: Kevin</p>
+      <p>Hometown: Shanghai, China</p>
+      <p>Education: University of Michigan-Ann Arbor LSA</p>
+      <p>Major: BS in Computer Science & BA in Economics</p>
+      <p>Hobbies: Listening to music, watching movies, playing soccer</p>
+      <p>Favourite soccer team: Manchester United</p>
+      <p>Click here to get a fact about me:</p>
+      <button id="fact" onclick="getRandomFact()">Click</button>
+      <div id="fact-container">
+        <form action="/form-handler" method="POST">
+        </div>
+      </div>
+      <div id="projects">
+     <h2>SKILLS: </h2>
+     <img src="Image/c++.jpg" width="140" height="140">
+     <img src="Image/python.jpg" width="140" height="140">
+     <img src="Image/java.jpg" width="140" height="140">
+     <img src="Image/git.jpg" width="140" height="140>"     
+     <br/><br/><br/>
+     <img src="Image/html.jpg" width="140" height="140">
+     <img src="Image/css.jpg" width="140" height="140">
+     <img src="Image/javascript.jpg" width="140" height="140">
+     <img src="Image/ms.jpg" width="140" height="140">
+     <h2>PROJECTS: </h2>
+     <img src="Image/md.jpg" width="300" height="250" />
+    <img src="Image/e.jpg" width="300" height="250" />
+    </div>
+      <div id="myprofile">
+      <h2>PROFILE: </h2>
+      <p>Click <a href="https://github.com/kunchen7">here</a> to see my GitHub profile!</p>
+      <p>I have not finishing my Linkedin Profile, but I will update it immediately when my profile is ready!</p>
+      <h2>EXPERIENCE: </h2>
+      <p>Google Software Product Sprint Participant@2022</p>
+      <p>Investment Analyst Intern at Haitong Securities@2021</p>
+      <p>TOEFL TA at Qianshi Education@2020</p>
+      <img src="Image/googlelogo.jpg" width="200" height="200">
+      <img src="Image/haitong.jpg" width="200" height="200">
+      <img src="Image/toefl.jpg" width="200" height="200">
+      <br/>
+    <h2>RESUME: </h2>
+      <div id="resume">
+      <button id="resume"><a href="Kun,Chen.pdf" style="text-decoration: none;" download>Download Resume</a></button>
+    </div>
+        </div>
+        <div id="contact">
+        <h2>CONTACT ME: </h2>
+        <form action="/form-handler" method="POST">
+        Name: <br/><input type="text" name="name-input" placeholder="Name"/>
+        <br/><br/>
+        Email:<br/><input type="text" name="email-input" placeholder="Email"/>
+        <br/><br/>
+        Contacting Reason: <select name = "wc-input">
```

Comment:
I'd stay consistent with the newlines you're using on Name and Email. Name and Email both put a newline after the description text. I think the reason this part of the page takes up a lot of space is that there's a lot of space between each box. It looks like you're using two <br>s in a row to put space between name and email. An alternative approach might be to put each in a div or a span and just increase the padding/margin at the bottom of each. Something like this:

HTML:
```
<div class="contact-input">
    Name: <br/><input type="text" name="name-input" placeholder="Name"/>
</div>
```

CSS
```
.contact-input {
    padding-bottom: 15px;
}
```